app/jobs/shipit/destroy_stack_job.rb

```diff
@@ -22,8 +22,15 @@ def perform(stack)
       commit_deployments_ids = Shipit::CommitDeployment.where(task_id: tasks_ids).pluck(:id)
       Shipit::CommitDeploymentStatus.where(commit_deployment_id: commit_deployments_ids).in_batches(&:delete_all)
       Shipit::CommitDeployment.where(id: commit_deployments_ids).in_batches(&:delete_all)
-      Shipit::Status.where(commit_id: commits_ids).in_batches(&:delete_all)
-      Shipit::Commit.where(id: commits_ids).in_batches(&:delete_all)
+
+      Shipit::Status.where(commit_id: commits_ids).find_in_batches do |batch|
+        Shipit::Status.where(id: batch.map(&:id)).delete_all
+      end
+
+      Shipit::Commit.where(id: commits_ids).find_in_batches do |batch|
```

Comment:
Any reason why we're doing a `Shipit::Commit.where(commit_id: commits_ids)` again rather than looping through the commits_id itself in batches?

Reply:
Nope, whoops! Fixed.